example/requirements.txt

```diff
@@ -0,0 +1,5 @@
+uvicorn
+starlette[full]
```

Comment:
Let's use the more explicit case instead. I think it's just...

```
starlette
jinja2
```

Or have I missed something?